court_scraper/base/captcha_solver.py

```diff
@@ -0,0 +1,65 @@
+from anticaptchaofficial.recaptchav2proxyless import *
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+import time
+
+# TODO: Support more captcha types.
+
```

Comment:
This class provides a library of options for solving captchas. So far, only recaptca_V2 is supported because it is needed for Wisconsin, but it was written in a way that assumes supporting more options through the entrypoint .solve.